pkg/capture/flow.go

```diff
@@ -223,30 +223,31 @@ func (f *FlowLog) Rotate() *hashmap.AggFlowMap {
 }
 
 func (f *FlowLog) transferAndAggregate() (agg *hashmap.AggFlowMap) {
+
+	// Initialize aggregate flow map / result
 	agg = hashmap.NewAggFlowMap()
 
-	// Reusable key conversion buffers
+	// Create reusable key conversion buffers
 	keyBufV4, keyBufV6 := types.NewEmptyV4Key(), types.NewEmptyV6Key()
-	var goDBKey types.Key
 
 	for k, v := range f.flowMap {
 
-		// Populate key buffer according to source flow
-		if v.isIPv4 {
-			keyBufV4.PutAllV4(v.epHash[0:4], v.epHash[16:20], v.epHash[32:34], v.epHash[36])
-			goDBKey = keyBufV4
-		} else {
-			keyBufV6.PutAllV6(v.epHash[0:16], v.epHash[16:32], v.epHash[32:34], v.epHash[36])
-			goDBKey = keyBufV6
-		}
+		// Check if the flow actually has any interesting information for us, otherwise
+		// delete it from the FlowMap
+		if v.packetsRcvd > 0 || v.packetsSent > 0 {
 
```

Comment:
I removed the `HasBeenIdle()` because I saw that for some reason the compiler did not inline this trivial check. Since it is only used once (here) and I added a comment I think that's acceptable.